Automatic Validation ended with: 
> 2026-05-02T02:19:06.6925274Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [~9.0~2025.08, ~9.0~2025.08]]

(Automated response - build 1288.)

--- a/manifests/c/Coq/CoqPlatform/2025.08.3/Coq.CoqPlatform.installer.yaml
+++ b/manifests/c/Coq/CoqPlatform/2025.08.3/Coq.CoqPlatform.installer.yaml
@@ -13,7 +13,6 @@ ProductCode: Rocq-Platform~9.0~2025.08
 AppsAndFeaturesEntries:
 - DisplayName: Rocq-Platform~9.0~2025.08
   Publisher: The Rocq Team
-  DisplayVersion: ~9.0~2025.08
   ProductCode: Rocq-Platform~9.0~2025.08
 InstallationMetadata:
   DefaultInstallLocation: C:\Rocq-Platform~9.0~2025.08

